app/views/discovered_hosts/_discovered_host.html.erb

```diff
@@ -1,9 +1,12 @@
 <td class="hidden-tablet hidden-xs display-two-pane"><%= link_to h(host.name), discovered_host_path(host) %></td>
-<td><%= model_name host %></td>
-<td><%= host.ip %></td>
-<td><%= discovery_attribute(host, :cpu_count) %></td>
-<td><%= number_to_human_size(discovery_attribute(host, :memory, 0) * 1024 * 1024) %></td>
+<td class="hidden-tablet hidden-xs"><%= model_name host %></td>
```

Comment:
The `hidden` classes don't disappear with two-pane so they either need to be on the headers too or remove them completely.
I prefer to remove them because it makes the two-pane look better but if you want to keep them for resize this needs to be re-added to the headers.